Makefile

```diff
@@ -1,41 +1,80 @@
 SHELL := /bin/bash
+include functions.mk
+
+# Name of the exporter
+EXPORTER_NAME := ebs-iops-reporter
+# valid: deployment or daemonset
+# currently unused
+EXPORTER_TYPE := deployment
 
 # All of the source files which compose the monitor. 
 # Important note: No directory structure will be maintained
 SOURCEFILES ?= monitor/main.py monitor/start.sh
```

Comment:
Sources will be different per exporter, maybe `monitor/$EXPORTER_NAME/{main.py,start.sh}`?